src/main/java/seedu/contax/logic/commands/EditTagCommand.java

```diff
@@ -18,8 +18,8 @@
 public class EditTagCommand extends Command {
     public static final String COMMAND_WORD = "edittag";
 
-    public static final String MESSAGE_USAGE = "`" + COMMAND_WORD + "`: **Edits the details of the specified tag.**\n"
-            + "Parameters: *INDEX t/NEW_TAGNAME*\n"
+    public static final String MESSAGE_USAGE = "`" + COMMAND_WORD + "`: **Edits the details of the specified tag.**"
+            + "\nParameters: *INDEX t/NEW_TAGNAME*\n"
             + "Example: `" + COMMAND_WORD + " 1 " + "t/Prospective Clients`";
 
     public static final String MESSAGE_EDIT_TAG_SUCCESS = "Edited Tag: %s";
```

Comment:
Similarly, there should be a break line after the colon here.

Reply:
thanks for the catch, updated in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/248/commits/2b67c04764a53ad085721a659134752196fea472